JPS_Ontology/ontology/ontosealevel/OntoSeaLevel.csv

```diff
@@ -0,0 +1,34 @@
+CulturalSite,Class,,,,,,A place or building of cultural importance.,https://www.theworldavatar.com/kg/ontosealevel/,Cultural site
+SeaLevelMeasurement,Class,,,,,,Sea level measurement.,https://www.theworldavatar.com/kg/ontosealevel/,Sea level measurement
+ActualSeaLevelMeasurement,Class,SeaLevelMeasurement,IS-A,,,,Actual sea level measurement.,https://www.theworldavatar.com/kg/ontosealevel/,Actual sea level measurement
+ForecastSeaLevelMeasurement,Class,SeaLevelMeasurement,IS-A,,,,Forecasted sea level measurement based on a forecasting model.,https://www.theworldavatar.com/kg/ontosealevel/,Forecast sea level measurement
+ForecastSeaLevelMeasurement,Class,https://www.theworldavatar.com/kg/ontotimeseries/Forecast,IS-A,,,,Forecasted sea level measurement based on a forecasting model.,https://www.theworldavatar.com/kg/ontosealevel/,
+Tree,Class,CulturalSite,IS-A,,,,Tree,https://www.theworldavatar.com/kg/ontosealevel/,Tree
+HeritageTree,Class,Tree,IS-A,,,,A tree that is the natural heritage of a specific place.,https://www.theworldavatar.com/kg/ontosealevel/,HeritageTree
+https://schema.org/Museum,Class,CulturalSite,IS-A,,,,,https://www.theworldavatar.com/kg/ontosealevel/,Museum
+https://schema.org/TouristAttraction,Class,CulturalSite,IS-A,,,,,https://www.theworldavatar.com/kg/ontosealevel/,Tourist attraction
+https://schema.org/LandmarksOrHistoricalBuildings,Class,CulturalSite,IS-A,,,,,https://www.theworldavatar.com/kg/ontosealevel/,Landmarks or historical building
+Monument,Class,CulturalSite,IS-A,,,,A structure for the commemoration of a person or event.,https://www.theworldavatar.com/kg/ontosealevel/,Monument
+Monument,Class,https://raw.githubusercontent.com/beAWARE-project/ontology/master/beAWARE_ontology#Monument,EQUIVALENT-TO,,,,A structure for the commemoration of a person or event.,https://www.theworldavatar.com/kg/ontosealevel/,Monument
+Scenario,Class,,,,,,Scenario that can act as input to a forecasting level for forecasting sea level.,https://www.theworldavatar.com/kg/ontosealevel/,Scenario
+SharedSocioEconomicPathway,Class,Scenario,IS-A,,,,A climate change scenario of projected socioeconomic changes.,https://www.theworldavatar.com/kg/ontosealevel/,Shared Socioeconomic Pathway
+measuredAt,Object Property,,,LocalMeanSeaLevel,http://www.opengis.net/ont/geosparql#Feature,exactly 1,A relationship between the local mean sea level and the reference point for the local mean sea level measurement.,https://www.theworldavatar.com/kg/ontosealevel/,measured at
+measuredIn,Data Property,,,SeaLevelMeasurement,http://www.w3.org/2001/XMLSchema#gYear,,A relationship between a sea level measurement and the time of measurement.,https://www.theworldavatar.com/kg/ontosealevel/,measured in
+measuredIn,Data Property,,,SeaLevelMeasurement,http://www.w3.org/2001/XMLSchema#date,,A relationship between a sea level measurement and the time of measurement.,https://www.theworldavatar.com/kg/ontosealevel/,measured in
```

Comment:
Remember, making two separate statements about an entity is interpreted as a logical _conjunction_. In this case here, that would imply that the range of `measuredIn` has to be both a year and a date (which is impossible). In order to achieve what you want, you need a single definition, with the range being the union of year and date.